sormas-backend/src/main/java/de/symeda/sormas/backend/common/StartupShutdownService.java

```diff
@@ -546,8 +556,14 @@ private void createOrUpdateDefaultUser(Set<UserRole> userRoles, String username,
 
 		User existingUser = userService.getByUserName(username);
 
+		boolean allUserRolesExist = !CollectionUtils.isEmpty(userRoles) && !userRoles.stream().noneMatch(Objects::isNull);
```

Comment:
I think the `!` has to be removed as well as now it will only be true if one of the values is `null`

Reply:
@vidi42 No, since we want to know if it's not empty so that 'allUserRolesExist'

Reply:
my bad